README.md

```diff
@@ -42,19 +67,20 @@ Also if you are using Ubuntu you can use PPA in order to install `tezos-client`.
 In order to do that run the following commands:
 ```
 sudo add-apt-repository ppa:serokell/tezos && sudo apt-get update
-sudo apt-get install tezos-client
+sudo apt-get install tezos-client-mainnet
+sudo apt-get install tezos-client-babylonnet
 ```
 
 ## Fedora (Red Hat) usage
 
-Download `.rpm` file from the CI and double-click on it or run:
+Build or download `.rpm` file from the CI and double-click on it or run:
 ```
 sudo yum localinstall <path to the rpm file>
 ```
 
 ## Other Linux distros usage
 
-Download statically built `tezos-client` binary from the CI.
+Download or build static `tezos-client` binary from the CI.
```

Comment:
Probably it would be better to say "Build static `tezos-client` binary or download it from the CI".